src/arrayMethodSort.js

```diff
@@ -5,7 +5,37 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let sortFunction = compareFunction;
+
+    if (!sortFunction) {
+      sortFunction = function(firstValue, secondValue) {
+        if (String(firstValue) > String(secondValue)) {
+          return 1;
+        } else if (String(firstValue) < String(secondValue)) {
+          return -1;
+        }
+
+        return 0;
+      };
+    }
```

Comment:
it's better move this logic to an own function and use as default function parameter 

```
const compareAsStrings = (a, b) => {
  const stringA = String(a);
  const stringB = String(b);
  
  if (stringA > stringB) {
    return 1;
  }
  
  if (stringA === stringB) {
    return 0;
  }
    
  return -1;
};
```